gulp/rsync.js

```diff
@@ -3,20 +3,45 @@ var gulp = require('gulp'),
     gutil = require('gulp-util'),
     rsync = require('gulp-rsync'),
     optional = require('optional'),
-    syncCloudConfig = optional('./cloud.conf.json') || {},
+    syncCloudConfig = optional('./cloud.conf.json'),
     syncCloudID = gutil.env['sync-cloud'] || 'default',
-    syncClouds = syncCloudConfig.clouds || {},
-    syncCloud = syncClouds[syncCloudID];
+    noSync = gutil.env['no-sync'] || false,
+    syncClouds = syncCloudConfig ? syncCloudConfig.clouds : {},
+    syncCloud = syncClouds ? syncClouds[syncCloudID] : undefined,
+
+    skypeRsync = 'Skypping rsync.';
```

Comment:
I'm not a native english speaker but I believe it should be `skipRsync` not `skipeRsync`.